kratos/geometries/triangle_3d_6.h

```diff
@@ -465,54 +465,49 @@ template<class TPointType> class Triangle3D6
 		return MathUtils<double>::Norm3(vx);
     }
 
-    /** This method calculates and returns area or surface area of
-     * this geometry depending to it's dimension. For one dimensional
-     * geometry it returns zero, for two dimensional it gives area
+    /** 
+     * @brief This method calculates and returns area or surface area of this geometry depending to it's dimension. 
+     * @details For one dimensional geometry it returns zero, for two dimensional it gives area
      * and for three dimensional geometries it gives surface area.
-     *
-     * @return double value contains area or surface
-     * area.
+     * @return double value contains area or surface area
      * @see Length()
      * @see Volume()
      * @see DomainSize()
      */
-    /**
-     * :TODO: could be replaced by something more suitable
-     * (comment by janosch)
-     */
     double Area() const override
     {
-        //return fabs( DeterminantOfJacobian( PointType() ) ) * 0.5;
-        // Approximation to avoid errors. Can be improved.
+        Vector temp;
```

Comment:
Check this @KratosMultiphysics/technical-committee. IMO, it makes sense, previous implementation was assuming that the nodes in the mid of the edges were aligned with the vertex nodes. This is more general.

Reply:
@KratosMultiphysics/technical-committee agrees with this change but suggests to change the temp to `array_1d`

Reply:
> @KratosMultiphysics/technical-committee agrees with this change but suggests to change the temp to `array_1d`

This can't be done as the size is not known at compile time. Resolving.

Reply:
We are in triangle 6 and we know the default integration method. So we can hardcode the size. I think for such a low level function, such a performance improvement can be justified.

Reply:
In that case we can do the same for any other integration. I would do in an independent PR, this PR is already old and it would make even more confusing the changes